charts/prometheus-operator/ci/certmanager-values.yaml

```diff
@@ -0,0 +1,3 @@
+admissionWebhooks:
+  certManager:
+    enabled: true
```

Comment:
this should be disabled by default.
not every user will have cert-manager installed.

Reply:
Yes, it is disabled by default. The `ci` files is just used by `ct` during the chart testing. It will do lint and install for every `*-values.yaml` in the `ci` directory.

However it does cause a problem currently because the Github actions spins up a kind cluster for deploying the chart. Deploying the chart with certmanager enabled but without certmanager deployed _is_ a problem - hence the failure from the Github workflow.